### DISCUSSION TITLE
Feature request: Direct shortcuts to organisation repos

### DISCUSSION THREAD

**Discussion by plastikfan** [community/community#27608] [Other Feature Feedback, Questions, & Ideas]
At the moment accessing organisation level respos (as opposed to repos directly under your username requires too much link clicking and is clumsy. I think this could be streamlined especially for commonly used repos. The pinned repos for an oragnisation does help in this regard, but still requires  too much link clicking.

What I'd like to be able to do is to be able to click the down arrow on my username icon on the top right of most github pages and click a link directly off this dropdown. Currently, this drop contains __Set Status__, __Your repositories__, __Your codespaces__ etc. I would like to be able to pin any repo of any organisation to this dropdown list, to allow me direct access to any repo of my choice. Alternatively, the drop could be automatically populated with the most recent 5 repos contributed to. This number could be configurable under account settings. This list is only the repos to which you have write access to, to avoid cluttering the list with other repos you may just be browsing.

If its seen as not being desireable to add more entries to that dropdown, could we have another dropdown specically for this purpose.

**Comment by jfullstackdev** [comment]
> If its seen as not being desireable to add more entries to that dropdown, could we have another dropdown specically for this purpose.

yes, the new List on the Stars tab, you can create a list named, say, My Pinned Repos, and add repos to that those you want to pin

**Reply by plastikfan** [reply]
Argh ok, but it still is one level of indirection away. I'd like a single direct access to a repo for fast access and also not mixed up with stars as that is for other repos

**Reply by jfullstackdev** [reply]
you can always bookmark that, as for me, I'm not closing tabs that are very important pages as my productivity technique or even if I close them, they are bookmarked and always one click away

**Reply by plastikfan** [reply]
I see what you're saying, but that kind of misses the point. Thats really a sticking plaster approach, where as I would prefer a better underlying UX. I mean really, its not that important, its just a suggestion to improve the UI.

**Reply by jfullstackdev** [reply]
yup, it's a request but if in case that will be implemented in the future, this is the workaround for now
